### PR TITLE
add validator profile for lunc community italia

### PR DESCRIPTION
**Pull request by LuncCommunityItalia**
add validator profile for lunc community italia

### CODE REVIEW

File: profile.json [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-    "discord": "anic1530 | 8moon#5925"
+    "email": "info@luncitalia.com",
+    "telegram": "@LUNCCommunityitalia",
+    "twitter": "@LUNC_Italia"
   },
    "notifications": {
-       "email": "anic1530@gmail.com"
+       "email": "info@luncitalia.com"
   }
 }
```
Comment (terran6) [inline profile.json:10]: Hello!  Please update your `profile.json` in your validator directory:

https://github.com/terra-money/validator-profiles/blob/master/validators/terravaloper1ef4hd8agtx8unaut0fc99zs9928qzm6txjcx40/profile.json

**Review by terran6 — CHANGES_REQUESTED** [review]

File: profile.json (hunk 1 above)
Comment (LuncCommunityItalia) [reply profile.json:10]: Ok, i have updated it

Comment (terran6) [reply profile.json]: > Ok, i have updated it

@LuncCommunityItalia Just delete the `profile.json` file outside of your validator directory and you should be good :)

Screen Shot 2022-11-04 at 12 11 50 PM

Comment (Man12C) [reply profile.json]: ok

**Comment by terran6** [comment]
@LuncCommunityItalia Do you mind resolving the conflicts?  After that, you should be good to go!

**Review by terran6 — APPROVED** [review]
👍🏼

**Pull request merged**